externs/shaka/offline.js

```diff
@@ -87,6 +88,8 @@ shaka.extern.StoredContent;
  *   The Streams that are stored.
  * @property {!Array.<string>} sessionIds
  *   The DRM offline session IDs for the media.
+ * @property {!Array.<Uint8Array>} initDatas
+ *   The DRM offline initData's for the media.
```

Comment:
They are only when the pssh is in the manifest, if the pssh is in the init segment, `drmInfo.initData` is empty.

Reply:
I think I might prefer it if we push those initDatas into drmInfo.initData instead.  I can't think of a way that would hurt anything, and it would help us avoid a change to the offline database format.

Reply:
Ok I'm going to do some tests and I will implement what you say.

Reply:
@joeyparrish , I just saw that there is a specific comment that says it is not necessary to save this information: https://github.com/google/shaka-player/blob/master/lib/offline/storage.js#L1011-L1014
Do you know why it is?